src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1303,6 +1303,40 @@ private bool CanAddToGroup(object parameters)
             return false;
         }
 
+        private void SelectUpstream(object parameters)
+        {
+            NodeModel.SelectUpStreamNeighbours();
+        }
+
+        private bool CanSelectUpstream(object parameters)
+        {
+            //Add checks for can stream
+            return true;
```

Comment:
if this just needs to return true all the time, there probably isnt any reason to implement it